include/objects.h

```diff
@@ -71,6 +71,9 @@ bool pop_outstanding_request(PgSocket *client, char *types, bool *skip);
 bool clear_outstanding_requests_until(PgSocket *server, char *types) _MUSTCHECK;
 bool queue_fake_response(PgSocket *client, char request_type) _MUSTCHECK;
 
+PgGlobalUser * find_exist_or_add_new_global_user(const char *name, const char *passwd) _MUSTCHECK;
+PgCredentials * find_exist_or_add_new_global_credentials(const char *name, const char *passwd) _MUSTCHECK;
```

Comment:
nit let's name these functions as follows:

```suggestion
PgGlobalUser * find_or_add_new_global_user(const char *name, const char *passwd) _MUSTCHECK;
PgCredentials * find_or_add_new_global_credentials(const char *name, const char *passwd) _MUSTCHECK;
```

Reply:
Done. Thanks!